iGeom/iGeom.cpp

```diff
@@ -1454,8 +1454,15 @@ CGMTagManager::CGMTagManager()
     // get the tag number for CATag
   DLIList<int> tag_types;
   int max_type = 0;
-  CubitAttribManager *cam = CGMApp::instance()->attrib_manager();
+
+#ifdef CUBIT_17_PLUS
```

Comment:
In line with my comment above, I don't think I used an ideal name for this compiler definition. Something like `SDK_17_PLUS` would be more accurate I think.

Reply:
`iGeom.h` now uses `CUBIT_VERSION_MAJOR` which is included from `CubitVersion.h`. This means we no longer have to add a preprocessor definition for it.